website/templates/emails/tou_notif.html.mako

```diff
@@ -0,0 +1,17 @@
+<%inherit file="notify_base.mako" />
+
+<%def name="content()">
+<tr>
+  <td style="border-collapse: collapse;">
+    Hello ${fullname},<br>
+    <br>
+    On August 10, 2020 the COS Websites and Services Terms of Use will change. The updates to the Terms are necessary to support continued use of the Websites and Services by the public.<br>
```

Comment:
Should we link to the [diff](https://github.com/CenterForOpenScience/cos.io/pull/1025/files)?